src/Form/ThematicCommunityMembershipFilterType.php

```diff
@@ -0,0 +1,170 @@
+<?php
+
+namespace App\Form;
+
+use App\Entity\JobEnum;
+use App\Entity\ThematicCommunity\ThematicCommunity;
+use App\Entity\ThematicCommunity\ThematicCommunityMembership;
+use App\Entity\UserListDefinition;
+use App\Entity\UserListDefinitionEnum;
+use App\ThematicCommunity\ThematicCommunityMembershipFilter;
+use Doctrine\ORM\EntityRepository;
+use Symfony\Bridge\Doctrine\Form\Type\EntityType;
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\Extension\Core\Type\ChoiceType;
+use Symfony\Component\Form\Extension\Core\Type\CountryType;
+use Symfony\Component\Form\Extension\Core\Type\DateType;
+use Symfony\Component\Form\Extension\Core\Type\HiddenType;
+use Symfony\Component\Form\Extension\Core\Type\IntegerType;
+use Symfony\Component\Form\Extension\Core\Type\TextType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+class ThematicCommunityMembershipFilterType extends AbstractType
+{
+    public function buildForm(FormBuilderInterface $builder, array $options)
+    {
+        $builder
+            ->add('firstName', TextType::class, ['required' => false])
+            ->add('lastName', TextType::class, ['required' => false])
+            ->add('gender', GenderType::class, [
+                'placeholder' => 'Tous',
+                'expanded' => true,
+                'required' => false,
+            ])
+            ->add('ageMin', IntegerType::class, ['required' => false])
+            ->add('ageMax', IntegerType::class, ['required' => false])
+            ->add('city', TextType::class, ['required' => false])
+            ->add('country', CountryType::class, ['required' => false])
+            ->add('role', ChoiceType::class, [
+                'required' => false,
+                'choices' => [
+                    'Adhérent',
+                    'Animateur local',
+                    'Porteur de projets citoyens',
```

Comment:
what about using `key => label` structure to be able to modify the only labels in the future and do not touch the keys?